developer/src/kmc/build.sh

```diff
@@ -44,9 +44,10 @@ if builder_start_action clean; then
   builder_finish_action success clean
 else
   # We need the schema file at runtime and bundled, so always copy it for all actions except `clean`
-  mkdir -p "$THIS_SCRIPT_PATH/build/src/"
-  cp "$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/techpreview/ldml-keyboard.schema.json" "$THIS_SCRIPT_PATH/build/src/"
-  cp "$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/techpreview/ldml-keyboardtest.schema.json" "$THIS_SCRIPT_PATH/build/src/"
+  mkdir -p "$THIS_SCRIPT_PATH/build/src/util/"
+  cp "$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/techpreview/ldml-keyboard.schema.json" "$THIS_SCRIPT_PATH/build/src/util/"
+  cp "$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/techpreview/ldml-keyboardtest.schema.json" "$THIS_SCRIPT_PATH/build/src/util/"
+  cp "$KEYMAN_ROOT/common/schemas/kvks/kvks.schema.json" "$THIS_SCRIPT_PATH/build/src/util/"
```

Comment:
So... it seems like the `NodeCompilerCallbacks.ts` utils need to be stored locally to the module itself?

I can understand needing a copy somewhere in the Developer build output - the files _would_ need to get bundled with everything else for distribution, after all.  Just... a small bit surprised they're not being located in `build/resources` or the like.  Not that it's likely to be a major issue.

Reply:
I try to make the compiled output have the same file layout as the distribution. Fewer surprises.